### PR TITLE
Add .sln files to gitignore

### PR DESCRIPTION
**Pull request by Thraka**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,4 @@ ehthumbs_vista.db
 
 [Bb]in/
 [Oo]bj/
+*.sln
```
Comment (tdykstra) [inline .gitignore]: Looks like there may be a missing newline character at the end of the file.

**Review by tdykstra — APPROVED** [review]

**Review by BillWagner — APPROVED** [review]
This LGTM. You can :shipit: when ready.

**Comment by Youssef1313** [comment]
@Thraka, Why .sln are added to gitignore?
The dotnet/samples readme states:

> Don't check in the solution file **if** it contains only one project.

That means solution files can be used if they have multiple related projects.

**Comment by tdykstra** [comment]
When needed, you can force check-in the .sln file.

**Comment by Youssef1313** [comment]
@tdykstra Thanks. Didn't know it's possible to do that.

**Pull request merged**